analysis_rsp.tex

```diff
@@ -0,0 +1,50 @@
+
+\subsubsection{Portal}
+
+\textbf{P-101}: User interface cone search on the object catalog, review the results overplotted on the HIPS map.
+
+\textbf{P-102}: Astronomical Data Query Language (ADQL) cone search on the object catalog with magnitude constraints, create a color-magnitude diagram in the results view.
```

Comment:
Astronomical Data Query Language (ADQL) search with column constraints, and creating plots based on the retrieved data in the results view.